definitions.yml

```diff
@@ -247,3 +247,34 @@ definitions:
         type: string
       body:
         type: string
+  Idps:
```

Comment:
What do you think to make this entities strictly related to Spid?

`Idps` -> `SpidIdps`
`IdpEntry` -> `SpidIdp`

Reply:
Yep, sounds good! I will update the pagopa/io-app#3008 too after this rename